AI Kit name changes for  Getting Started AI Samples (#2028)
oneapi-src/oneAPI-samples: AI-and-Analytics/Getting-Started-Samples/IntelTensorFlow_GettingStarted/README.md

This sample code shows how to get started with TensorFlow*. It implements an example neural network with one convolution layer and one ReLU layer. You can build and train a TensorFlow* neural network using a simple Python code. Also, by controlling the build-in environment variable, this sample attempts to demonstrate explicitly how Intel® oneDNN Primitives are called and shows their performance during the neural network training.

## Prerequisites

| Optimized for          | Description
|:---                    |:---
| OS                     | Ubuntu* 18.0.x (and newer) <br> Windows* 10 
| Hardware               | Intel® Xeon® Scalable processor family
| Software               | TensorFlow

> TensorFlow* is ready for use once you finish the Intel AI Tools installation. You can refer to the oneAPI [product page](https://software.intel.com/en-us/oneapi) for tools installation and the *[Get Started with the Intel® AI Tools for Linux*](https://software.intel.com/en-us/get-started-with-intel-oneapi-linux-get-started-with-the-intel-ai-analytics-toolkit)* for post-installation steps and scripts.

Reviewer: ```suggestion
Before running the sample, install the Intel Extension for TensorFlow* via the Intel AI Tools Selector or Offline Installer. Refer to *[Get Started with the Intel® AI Tools for Linux*](https://software.intel.com/en-us/get-started-with-intel-oneapi-linux-get-started-with-the-intel-ai-analytics-toolkit)* for post-installation steps and scripts.
```

Author: No ITEX involved in this sample.

For details, refer to the *[Quick Example on Intel CPU and GPU](https://intel.github.io/intel-extension-for-tensorflow/latest/examples/quick_example.html)* topic of the *Intel® Extension for TensorFlow** documentation. 

> ## Run the Sample Locally (Only applicable to AI Tools Offline Installer)

Reviewer: ```suggestion
## Run the Sample Locally (Only applicable to Intel AI Tools Offline Installer)
```

> ## Run the Sample Locally (Only applicable to Intel AI Tools Offline Installer)

Reviewer 2: ```suggestion
### Steps for Intel AI Tools Offline Installer  
```
